sdk/security/keyvault/azadmin/backup/client_test.go

```diff
@@ -177,3 +181,48 @@ func TestBeginSelectiveKeyRestoreOperation(t *testing.T) {
 		time.Sleep(60 * time.Second)
 	}
 }
+
+func TestAPIVersion(t *testing.T) {
+	apiVersion := "7.3"
+	var requireVersion = func(t *testing.T) func(req *http.Request) bool {
+		return func(r *http.Request) bool {
+			version := r.URL.Query().Get("api-version")
+			require.Equal(t, version, apiVersion)
+			return true
+		}
+	}
```

Comment:
`t` is in scope and you don't need to capture it, so you can simplify this to
```suggestion
	requireVersion := func(r *http.Request) bool {
		version := r.URL.Query().Get("api-version")
		require.Equal(t, apiVersion, version)
		return true
	}
```